### PR TITLE
Android Spaces Screen Sharing example

### PR DESCRIPTION
**Pull request by nbirkenshaw-mux**
How to use screen sharing in the Android Spaces SDK. This code is the same as that developed during the upcoming guide on the subject.

This is a separate example from camera/microphone sharing because thanks primarily to the permissions difference there are quite significant integration differences which are far clearer when presented as separate examples.

### CODE REVIEW

**Comment by refactornator** [comment]
I'd like this to have a README for getting started with the example like the `android-spaces-real-time-app` example has.

And I'd like you to also add info about this example to the repo's main README like the other examples.

File: android-spaces-real-time-screen-sharing/app/src/main/java/com/mux/screenshareexample/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+        status.setText("Joining space");
+
+        final String jwt = "YOUR JWT";
+
+        SpaceConfiguration spaceConfiguration = null;
+
+        try {
+            spaceConfiguration = SpaceConfiguration.newBuilder()
+                    .setSignalsHost("wss://spaces.staging.mux.com")
```
Comment (refactornator) [inline android-spaces-real-time-screen-sharing/app/src/main/java/com/mux/screenshareexample/MainActivity.java:240]: We should take this out and use production instead.

File: android-spaces-real-time-screen-sharing/app/src/main/java/com/mux/screenshareexample/MainActivity.java [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        // For this to work the launchMode of MainActivity must be set to "singleTop" in AndroidManifest.xml
+        // This ensures Activity instances are unique and so the notification won't do things like launch a new activity
+        // If you have more complex use cases which require things like preserving a large back stack you will need to use one of the
+        // normal approaches for doing that, but it will be highly specific to your application.
+        Intent intent = new Intent(this, MainActivity.class);
+        PendingIntent pendingIntent = PendingIntent.getActivity(this, 1, intent, PendingIntent.FLAG_MUTABLE);
+
+        // The setSmallIcon is necessary or the operating system will silently disregard the notification, and so the whole screen sharing request
+        // setOngoing ensures the user can't remove the notification while screensharing
+        builder.setPriority(Notification.PRIORITY_DEFAULT)
```
Comment (refactornator) [inline android-spaces-real-time-screen-sharing/app/src/main/java/com/mux/screenshareexample/MainActivity.java:154]: I get deprecation warnings in Android studio:
`'setPriority(int)' is deprecated as of API 26: Android 8.0 (Oreo)`
and
`'PRIORITY_DEFAULT' is deprecated as of API 26: Android 8.0 (Oreo)`

**Comment by nbirkenshaw-mux** [comment]
README added and linked to from root README.

Also updated other Android readme to remove mention of beta status.

**Review by refactornator — APPROVED** [review]
LGTM 👍

**Pull request merged**